source/manual/datalabs-content-store.html.md

```diff
@@ -0,0 +1,52 @@
+---
+owner_slack: "#govuk-data-labs"
+title: Use Content Store
+section: Data Labs Team Manual
+layout: manual_layout
+parent: "/manual.html"
+---
+
+The [Content Store](https://docs.publishing.service.gov.uk/apps/content-store.html) is a MongoDB database of almost all the content published on GOV.UK.
+
+The Content Store does not have all GOV.UK content. The Content Store has the content itself, but does not include dynamic elements such as top links on taxon pages, navigation elements, or search result pages.
+
+Contact a GOV.UK Data Labs software developer through the [GOV.UK Data Labs slack channel](https://gds.slack.com/archives/CHR4UQKU4) for more information.
+
+Consider using the [GOV.UK mirror](link) if you need a more representative data source of what users actually see on the website.
+
+## Get access to the Content Store
+
+To access the Content Store, you download a copy of the production version of the Content Store and query that local version using Docker.
+
+1. Sign into AWS. See the [GOV.UK Developer Docs on getting AWS access](https://docs.publishing.service.gov.uk/manual/get-started.html#7-get-aws-access) for more information.
+
+1. Go to the [`govuk-integration-database-backups` AWS S3 bucket](https://s3.console.aws.amazon.com/s3/buckets/govuk-integration-database-backups?prefix=mongo-api%2F&region=eu-west-1) and then go to the `mongo-api` folder.
+
+1. Download a copy of the production version of the Content Store. The production version file is `{DATETIME}-content_store_production.gz`, where `{DATETIME}` is a date and time in `YYYY-MM-DDTHH:MM:SS` format.
+
+1. Run the following in the command line to extract the `content_items.bson` file from the `content_store_production` folder of the downloaded production version file:
```

Comment:
Interestingly, when i downloaded the file I found the date format to be `YYYY-MM-DDTHH_MM_SS`

Reply:
underscores instead of colons